backend/app.js

```diff
@@ -1833,6 +1858,7 @@ app.post('/api/tomp3', optionalJwt, async function(req, res) {
 });
 
 app.post('/api/tomp4', optionalJwt, async function(req, res) {
+    req.setTimeout(0); // remove timeout in case of long videos
```

Comment:
For large video files, the request timeout was often reached. Here, we remove it to prevent duplicate download requests.